packages/react/src/components/Authenticator/ResetPassword/ResetPassword.tsx

```diff
@@ -46,29 +48,30 @@ export const ResetPassword = (): JSX.Element => {
         });
       }}
     >
-      <Heading level={1}>{headerText}</Heading>
+      <Flex direction="column">
+        <Heading level={3}>{headerText}</Heading>
 
-      <Fieldset disabled={isPending}>
-        <Label data-amplify-resetpassword-label="">
-          <Text>{I18n.get('Username')}</Text>
-          <Input
+        <FieldGroup direction="column" disabled={isPending}>
+          <TextField
             autoComplete="username"
```

Comment:
This `autoComplete` value will depend on the primary login mechanism, right?

Reply:
Mmm I suppose it will, yes. So when we update the string value, we can update this too